plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -727,12 +729,182 @@ public void testTimestamp(ZoneId sessionZone)
                 .addRoundTrip("timestamp", timestampDataType(3).toLiteral(timeGapInKathmandu), TIMESTAMP_MILLIS, timestampDataType(3).toLiteral(timeGapInKathmandu))
                 // max value in Oracle
                 .addRoundTrip("timestamp", "TIMESTAMP '9999-12-31 00:00:00.000'", TIMESTAMP_MILLIS, "TIMESTAMP '9999-12-31 00:00:00.000'")
-                .execute(getQueryRunner(), session, oracleCreateAndInsert("test_timestamp"))
```

Comment:
Any specific for removing this execution option ?

Reply:
Previously it was there, because we supported only timestamp(3).
But because now we support any precision if we create just timestamp - default precision became 6.